packages/cli/src/commands/migrate/packageScripts.ts

```diff
@@ -41,6 +41,10 @@ export async function command() {
       if (scripts.start?.includes('--check')) {
         startCmd.push('--check');
       }
+      if (scripts.start?.includes('--disable-config-watching')) {
+        startCmd.push('--disable-config-watching');
+      }
```

Comment:
No need to add these, the old scripts don't have these flags

Reply:
Done!